lib/src/launchpad.rs

```diff
@@ -238,6 +248,255 @@ impl LaunchpadMk2 {
+                                       0x00,
+                                       0x20,
+                                       0x29,
+                                       0x02,
+                                       0x10,
+                                       0x28,
+                                       led.position,
+                                       led.color,
+                                       0xF7])
+                        .expect("Fail");
+    }
+
+    /// Set a single LED to a palette color. Use `light_single` instead, its faster.
+    pub fn light_led(&mut self, led: &ColorLed) {
+        // F0h 00h 20h 29h 02h 18h 0Ah <LED> <Colour> F7h
+        // Message can be repeated up to 80 times.
+        self.light_leds(&[led])
+    }
+
+    /// Set LEDs to a certain color. Up to 80 LEDs can be set uniquely at once.
+    pub fn light_leds(&mut self, leds: &[&ColorLed]) {
+        assert!(leds.len() <= 80);
+        for led in leds {
+            assert_position_pro(led.position);
+            assert_color(led.color);
+            self.output_port
+                .write_sysex(0,
+                             &[0xF0,
+                               0x00,
+                               0x20,
+                               0x29,
+                               0x02,
+                               0x10,
+                               0x0A,
+                               led.position,
+                               led.color,
+                               0xF7])
+                .expect("Fail");
+        }
+    }
+
+    /// Light a column of LEDs to the same color.
+    pub fn light_column(&mut self, col: &ColorColumn) {
+        // F0h 00h 20h 29h 02h 18h 0Ch <Column> <Colour> F7h
+        // Message can be repeated up to 9 times.
+        self.light_columns(&[col])
+    }
+
+    /// Light columns of LEDs to the same color. Each column may be set to a
+    /// unique color. Up to 9 columns may be set at once.
+    pub fn light_columns(&mut self, cols: &[&ColorColumn]) {
+        assert!(cols.len() <= 10);
+        for col in cols {
+            assert_column_pro(col.column);
+            assert_color(col.color);
+            self.output_port
+                .write_sysex(0,
+                             &[0xF0,
+                               0x00,
+                               0x20,
+                               0x29,
+                               0x02,
+                               0x10,
+                               0x0C,
+                               col.column,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
+                               col.color,
```

Comment:
I didn't see a "Light the column to a single color" call in the programming reference for the Launchpad Pro. To get a full column lit to a single color, you'd need to repeat the color 10 times. Supporting arbitrary color combinations seems like a good thing here, but changing `ColorColumn` to take a `vec` of `Color` seemed like it'd be potentially problematic for the MK2.